### ISSUE TITLE
Add Electrical Symbols

### ISSUE THREAD

**Issue by joel-jose** [Xaviju/inkscape-open-symbols#68]
Please add electrical symbols from:
1. https://upload.wikimedia.org/wikipedia/commons/6/6a/Electrical_symbols_library.svg
2. http://mirror.unl.edu/ctan/graphics/pgf/contrib/circuitikz/doc/circuitikzmanual.pdf (section 4.1 onwards)

There's high demand for this library: https://bugs.launchpad.net/inkscape/+bug/1533355

I had tried creating it by following the tutorial. But the symbols were not editable once added to the document.

**Comment by Xaviju** [comment]
Hi @joel-jose 
Would you mind making a pull request with your symbol set. It might be quite fast to fix the issue. I could help :)

**Comment by joel-jose** [comment]
I will do that. Thank you!

**Comment by Xaviju** [comment]
Hi @joel-jose 

- is there any chance I can get those icons in separate files each one with their name? Separating each icon in a single file would make the work lihtning fast.
- I was trying to separate it myself but I have no idea what name give to each of the symbols.

**Comment by joel-jose** [comment]
Hi @Xaviju, 

Apologies for the delay in responding. I have created separate files for a few of them: https://drive.google.com/file/d/0Bw3patRr3z0KdWRaNUhPVDBRbW8/view?usp=sharing (tried uploading the archive here, didn't work). Can you check if they are in a form you wanted? If yes, I will work on getting all of the symbols in separate files.

**Comment by Xaviju** [comment]
Hi @joel-jose and sorry for the late reply. 
I tried the attached file and yes, that would be perfect.

**Comment by joel-jose** [comment]
Hi @Xaviju, I have updated the archive with all the symbols: https://drive.google.com/file/d/0Bw3patRr3z0KN2MzbHVkclJLbFE/view?usp=sharing.
The folder structure is just for convenience. All symbols have unique identifiers.
Thank you!

**Comment by Xaviju** [comment]
Has been merged here https://github.com/Xaviju/inkscape-open-symbols/commit/319737dcac6e49b7988df73f046496bc975c49b6
Please test!
Thanks

**Comment by joel-jose** [comment]
This is perfect! Thanks a lot!

Just one concern: to edit the stroke width, the symbol has to be converted to a group by Edit>Clone>Unlink Clone. When I do that, the position and the size of the symbol on the canvas changes. This does not happen with other symbol sets. Any workarounds?

I will work on giving descriptive titles for the symbols. 

Thanks again!

**Comment by Xaviju** [comment]
Thanks for testing! 
Finally it was more work than expected so I might have created some bugs. I'll compare it with other symbols and check how to fix it.
I set the default stroke width to 2px to ensure that the symbol preview looks good. Please change it or remove it as you like.

**Comment by joel-jose** [comment]
Hi @Xaviju,
I have fixed the issue mentioned above by removing the `viewBox` attribute from symbols. I have also replaced the `title` tag with the text from `id`. For now, the ids are descriptive enough. I will definitely spend some time later improving the `title` texts. I am not used to GitHub, so I'll just share the updated SVG file here: https://drive.google.com/file/d/0Bw3patRr3z0KTlBsd2ZCWTRFajA/view?usp=sharing

Thanks!
